README.md

```diff
@@ -50,17 +52,21 @@ Several template variables are made available to the script running inside the t
 * `title` subject (%s)
 * `messageLines` array of body lines (%b)
 
-`dateFnsFormat` is the date-fns [format](https://date-fns.org/docs/format) function. See the [html-bootstrap](https://github.com/ariatemplates/git-release-notes/blob/master/templates/html-bootstrap.ejs) for sample usage.
+`dateFnsFormat` is the date-fns [format](https://date-fns.org/docs/format) function. See the [html-bootstrap](https://github.com/ariatemplates/git-release-notes/blob/master/templates/html-bootstrap.ejs) for usage example.
+
+`options` the object documented below. Useful for parsing the repository name. See the [html-bootstrap](https://github.com/ariatemplates/git-release-notes/blob/master/templates/html-bootstrap.ejs) for sample usage.
+
+`request` is an instance of [sync-request](https://www.npmjs.com/package/sync-request). This can be useful for querying the Jira API for example to extract extra metadata about a ticket related to a commit.
```

Comment:
I see your point on the ugliness of the intermediate step, but still I'm not convinced we should add logic in the template.

I'm thinking maybe we can implement something like [`mocha --require`](https://mochajs.org/#usage) or [`jest setupFiles`](https://facebook.github.io/jest/docs/configuration.html#setupfiles-array) so an option that allows you to inject your custom script. It would either add objects to you template or probably better it'll receive the `commits` array and return a modified version that can then be used in the template.